nf-test.config

```diff
@@ -3,14 +3,20 @@ config {
     testsDir "."
 
     // nf-test directory including temporary files for each test
-    workDir "/tmp"
-
-    // location of library folder that is added automatically to the classpath
-    libDir "tests/lib/"
+    workDir System.getenv("NFT_WORKDIR") ?: ".nf-test"
 
     // location of an optional nextflow.config file specific for executing tests
-    configFile "nextflow.config"
+    configFile "tests/nextflow.config"
 
     // run all test with the defined docker profile from the main nextflow.config
-    profile ""
+    profile "test"
+
+    // List of filenames or patterns that should be trigger a full test run
+    triggers 'nextflow.config', 'nf-test.config', 'conf/test.config', 'conf/test_full.config'
```

Comment:
no .nf files?

Reply:
the nf files are automatically detected by nf-test with the `--changed-since` feature in the CI

the triggers here meanwhile will make sure all tests are run if any of the trigger files are changed